From fef8dbbca07d5ab5358191e81575593fdd76c07a Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 29 Nov 2024 18:19:59 +0100
Subject: [PATCH 1514/1695] plat-stm32mp1: remove unused function dump_context

Remove the unused function dump_context().

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d761205087f4a10c078aafd0b8f556cabb861f09 plat-stm32mp1: remove unused function dump_context]
Fixes: cf483e4784e5 ("plat-stm32mp1: add low power drivers for stm32mp1")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423276
Change-Id: Iae93cbe254a6b7ac122e5c3d9e10955fdd50e879
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/437072
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/pm/context.c | 31 ------------------------
 1 file changed, 31 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/pm/context.c b/core/arch/arm/plat-stm32mp1/pm/context.c
index 6ea8cd941..85324b27c 100644
--- a/core/arch/arm/plat-stm32mp1/pm/context.c
+++ b/core/arch/arm/plat-stm32mp1/pm/context.c
@@ -153,37 +153,6 @@ static struct pm_mailbox *get_pm_mailbox(void)
 	return (struct pm_mailbox *)mailbox_base;
 }
 
-#if TRACE_LEVEL >= TRACE_DEBUG
-static void __maybe_unused dump_context(void)
-{
-	struct pm_mailbox *mailbox = get_pm_mailbox();
-	struct retram_resume_ctx *ctx = get_retram_resume_ctx();
-
-	clk_enable(pm_clocks.rtcapb);
-
-	DMSG("Backup registers: address 0x%" PRIx32 ", magic 0x%" PRIx32,
-	     *(uint32_t *)stm32mp_bkpreg(BCKR_CORE1_BRANCH_ADDRESS),
-	     *(uint32_t *)stm32mp_bkpreg(BCKR_CORE1_MAGIC_NUMBER));
-
-	clk_disable(pm_clocks.rtcapb);
-
-	clk_enable(pm_clocks.bkpsram);
-
-	DMSG("BKPSRAM mailbox:  0x%" PRIx32 ", zd 0x%" PRIx32 ", ep 0x%" PRIx32,
-	     mailbox->magic, mailbox->zq0cr0_zdata,
-	     mailbox->core0_resume_ep);
-
-	DMSG("BKPSRAM context:  teeram backup @%" PRIx32 ", resume @0x%" PRIx32,
-	     ctx->teeram_bkp_pa, ctx->resume_pa);
-
-	clk_disable(pm_clocks.bkpsram);
-}
-#else
-static void __maybe_unused dump_context(void)
-{
-}
-#endif
-
 /*
  * Save and restore functions
  */
-- 
2.47.2

